geoh5py/shared/concatenation.py

```diff
@@ -695,6 +695,20 @@ def parent(self, parent):
         if f"Property:{self.name}" not in parental_attr:
             parental_attr[f"Property:{self.name}"] = as_str_if_uuid(self.uid)
 
+    @property
+    def n_values(self) -> np.ndarray:
+        """Number of values in the data."""
+
+        n_values = None
+        depths = getattr(self.property_group, "depth_", None)
+        if depths:

```

Comment:
```suggestion
        if depths and depths is not self:
```

Reply:
doesn't work because the getattr results in a reference back to the n_values property.

Reply:
`n_values` of the Concatenated class yes?

Reply:
..ok, maybe it works after all... verifying

Reply:
done.